src/myapp.h

```diff
@@ -21,6 +21,9 @@ struct MyApp : wxApp {
         #ifdef __WXGTK__
             locale.AddCatalogLookupPathPrefix(L"/usr");
             locale.AddCatalogLookupPathPrefix(L"/usr/local");

```

Comment:
But the FHS paths are already hardcoded (even though they will not work due to missing `/locale` subdirectory).